templates/worker-router/README.md

```diff
@@ -1,26 +1,38 @@
-## Template: worker-router
+## Template: Worker-Router 
+
+This template demonstrates how to use the [`itty-router`](https://github.com/kwhitley/itty-router) package to implement routing in a Cloudflare Workers project.
+
+### Deploy with Workers
 
 [![Deploy with Workers](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com/?url=https://github.com/cloudflare/workers-sdk/tree/main/templates/worker-router)
 
-This template demonstrates using the [`itty-router`](https://github.com/kwhitley/itty-router) package to add routing to your Cloudflare Workers.
+### Prerequisites
+
+Before you begin, ensure you have `wrangler` CLI installed. You can install it using npm:
 
-[`index.js`](https://github.com/cloudflare/worker-template-router/blob/master/index.js) is the content of the Workers script.
+```sh
+npm install -g @cloudflare/wrangler
+```
 
-## Setup
+### Setup
 
-To create a `my-project` directory using this template, run:
+To create a new project directory using this template, run one of the following commands based on your package manager preference:
 
 ```sh
-$ npx wrangler generate my-project worker
+$ npx wrangler generate my-project worker-prospector
 # or
-$ yarn wrangler generate my-project worker
+$ yarn wrangler generate my-project worker-prospector
 # or
-$ pnpm wrangler generate my-project worker
+$ pnpm wrangler generate my-project worker-prospector
 ```
 
-Before publishing your code you need to edit `wrangler.toml` file and add your Cloudflare `account_id` - more information about configuring and publishing your code can be found [in the documentation](https://developers.cloudflare.com/workers/learning/getting-started).
+### Configuration
 
-Once you are ready, you can publish your code by running the following command:
+Before deploying your project, you need to configure the `wrangler.toml` file with your Cloudflare account details. Replace the `account_id` field with your Cloudflare account ID. Detailed instructions on configuring your project can be found in the [official Cloudflare Workers documentation](https://developers.cloudflare.com/workers/learning/getting-started).
```

Comment:
With Wrangler v2+ this is also not really needed any more. Wrangler is able to determine the account id from your user login.